index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio by Abenezer Samuel</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="toolbar">
+    <nav class="nav-items">
+      <h2 class="logo">Ebenezer</h2>
+      <img src="./image/menu.png" alt="menu bar image" />
+    </nav>
+  </header>
+  <section class="bg">
+    <section class="headline">
+    <h1 class="Primay-text">
+      I'm Abenezer.<br />
+      Glad to see <br />you!
+    </h1>
+    <p class="secondary-text">
+      I’m a software developer! I can help you build a product , feature or
+      website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <div class="social">
+      <h3 class="connect">LET'S CONNECT</h3>
+      <ul class="social-links">
+         <li>
+          <a href="https://github.com/abenisamuel11" target="_blank" rel="noopener noreferrer"><img
+              src="./image/Github.png" alt="github logo"></a>
+        </li>
+       <li>
+          <a href="https://www.linkedin.com/in/abenezer-samuel/" target="_blank" rel="noopener noreferrer"><img
+              src="./image/linkdin.png" alt="linkdin logo" /></a>
+        </li>
+        <li>
+          <a href="" target="_blank" rel="noopener noreferrer"><img src="./image/A.png" alt="angle list icon" /></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer"><img src="./image/twitter.png"
+              alt="twitter.icon" /></a>
+        </li>
+
+        <li>
+          <a href=" target="_blank" rel="noopener noreferrer"><img src="./image/medium.png" alt="medium logo" /></a>
+        </li>
+       
+
+        
+      </ul>
+    </div>
+    <br />
+  </section>
```

Comment:
- The document is missing the other closing `section` tag, `html`, and `body` tag. please, double-check the document to fix this by adding the missing closing corresponding tag to improve your document.